public/app/types/unified-alerting.ts

```diff
@@ -59,6 +59,7 @@ export interface AlertingRule extends RuleBase {
   totals?: Partial<Record<Lowercase<GrafanaAlertState>, number>>;
   totalsFiltered?: Partial<Record<Lowercase<GrafanaAlertState>, number>>;
   activeAt?: string; // ISO timestamp
+  folderUid?: string;
```

Comment:
This was on the object but not in the types. I think there are more shenanigans going on there that I didn't want to unravel for this, as it's a customer bug 😁

Reply:
This is correct. IIRC We have two types representing prometheus rules `PromRuleDTO` and `AlertingRule`
I've updated `PromRuleDTO` to include `folderUid` but forgot to update this one :)
Unfortunately, the types are shared between Grafana an vanilla Prom, so Grafana-specific fields needs to be optional
I think we could move `folderUid` to the `RuleBase` interface, as it should also be available for recording rules

Reply:
:+1: Thanks - I've moved it to RuleBase instead, makes sense